scripts/cleanup-sam-resources.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+#
+# A script for cleaning up a user's Sam resources, such as those not cleaned
+# up after testing. This will attempt to delete all controlled (both shared and
+# private) resources owned by a user in Sam, as well as all their workspaces.
+#
```

Comment:
Is it possible to put in a min_age guard, so that for example we don't delete anything less than 14 days old? That would leave any necessary resources around for test investigations, debugging, etc.